components/explore/SidebarFilter.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div :class="{ 'mr-4 bordered sticky': open }">
+    <NeoSidebar :reduce="false" :open="open" fullheight>
+      <b-collapse :open="false" animation="slide">
+        <template #trigger="props">
+          <div class="is-flex" role="button" :aria-expanded="props.open">
+            <p
+              :class="[
+                'card-header-title',
+                {
+                  'has-text-weight-semibold': props.open,
+                  'has-text-weight-normal': !props.open,
+                },
+              ]">
+              {{ $t('tabs.tabActivity.price') }}
+            </p>
+            <a class="card-header-icon">
+              <b-icon :icon="props.open ? 'minus' : 'plus'" />
+            </a>
+          </div>
+        </template>
+        <form class="p-4" @submit.prevent="setPriceRange">
+          <b-field label="MIN" label-position="inside">
+            <b-input
+              v-model="range.min"
+              type="number"
+              min="0"
+              step="any"
+              :placeholder="$t('query.priceRange.minPrice')"
+              data-cy="input-min" />
+          </b-field>
+          <b-field label="MAX" label-position="inside">
+            <b-input
+              v-model="range.max"
+              min="0"
+              step="any"
+              type="number"
+              :placeholder="$t('query.priceRange.maxPrice')"
+              data-cy="input-max" />
+          </b-field>
+          <NeoButton data-cy="apply" expanded @click.native="setPriceRange">
+            {{ $t('general.apply') }}
+          </NeoButton>
+        </form>
+      </b-collapse>
+      <b-collapse :open="true" animation="slide">
+        <template #trigger="props">
+          <div class="is-flex" role="button" :aria-expanded="props.open">
+            <p
+              :class="[
+                'card-header-title',
+                {
+                  'has-text-weight-semibold': props.open,
+                  'has-text-weight-normal': !props.open,
+                },
+              ]">
+              {{ $t('offer.status') }}
+            </p>
+            <a class="card-header-icon">
+              <b-icon :icon="props.open ? 'minus' : 'plus'" />
+            </a>
+          </div>
+        </template>
+        <div class="p-4">
+          <b-field>
+            <b-checkbox v-model="listed"> {{ $t('sort.listed') }} </b-checkbox>
+          </b-field>
+          <b-field>
+            <b-checkbox v-model="owned"> {{ $t('sort.own') }} </b-checkbox>
+          </b-field>
+        </div>
+      </b-collapse>
+    </NeoSidebar>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { NeoButton, NeoSidebar } from '@kodadot1/brick'
+
+const { $store, $consola } = useNuxtApp()
+const route = useRoute()
+const router = useRouter()
+const { decimals } = useChain()
+const range = ref({
+  min: Number(route.query?.min) / 10 ** decimals.value || 0,
```

Comment:
Dont we have some utils for that?

Reply:
nice catch! `toDecimals` & `fromDecimals`